### PR TITLE
added missing decorator to eigh in numpy frontend

### LINKED ISSUE #14942: eigh

### PR DESCRIPTION
**Pull request by kai271828**
Close #14942

### CODE REVIEW

File: ivy/functional/frontends/numpy/linalg/matrix_eigenvalues.py [hunk 1]
```diff
@@ -17,6 +17,6 @@ def eig(a):
     return ivy.eig(a)
 
 
-@from_zero_dim_arrays_to_scalar
+@to_ivy_arrays_and_back
```
Comment (AnnaTz) [inline ivy/functional/frontends/numpy/linalg/matrix_eigenvalues.py:20]: Thanks for adding the missing decorator, but I don't think you should be removing `@from_zero_dim_arrays_to_scalar`.

**Review by AnnaTz — CHANGES_REQUESTED** [review]
Since the function was already implemented I don't think the PR should be linked to the ToDo list.  Feel free to pick another item of the list to implement 🙂

**Comment by kai271828** [comment]
Hi, @AnnaTz . I want to ask, why are many functions already implemented but not tracked by an issue or marked as completed?

**Comment by AnnaTz** [comment]
> Hi, @AnnaTz . I want to ask, why are many functions already implemented but not tracked by an issue or marked as completed?

I guess people forget to link them. Feel free to make a comment under the ToDo list when you find an already implemented function so I can check it as done.

**Pull request merged**